libmamba/src/core/package_handling.cpp

```diff
@@ -354,14 +356,53 @@ namespace mamba
         LOG_INFO << "Extracting " << file << " to " << destination;
         extraction_guard g(destination);
 
+        struct archive* a;
+        int r;
```

Comment:
These variables can be initialized when defined (see below).

Reply:
+1